HWLesson 1.4/HWLesson 1.4/Program.cs

```diff
@@ -0,0 +1,111 @@
+            var countElements = Console.ReadLine();
+            if (int.TryParse(countElements, out int count) && count > 0)
+            {
+                int[] massiveElem = new int[count];
+
+                InputRandomElem(massiveElem);
+
+                string[] evenNumber = new string[CountEven(massiveElem)];
+                string[] oddNumber = new string[massiveElem.Length - CountEven(massiveElem)];
+                int evenIndex = 0;
+                int oddIndex = 0;
+
+                foreach (var i in massiveElem)
+                {
+                    if (i % 2 == 0)
+                    {
+                        evenNumber[evenIndex] = IntToEngAlphabet(i);
+                        evenIndex++;
+                    }
+                    else
+                    {
+                        oddNumber[oddIndex] = IntToEngAlphabet(i);
+                        oddIndex++;
+                    }
+                }
+
+                if (UpperCharCount(evenNumber) > UpperCharCount(oddNumber))
+                {
+                    Console.WriteLine($"Array 'evenNumber' has more uppercase letters ({UpperCharCount(evenNumber)}) then array 'oddNumber' ({UpperCharCount(oddNumber)})");
+                }
+                else if (UpperCharCount(evenNumber) < UpperCharCount(oddNumber))
+                {
+                    Console.WriteLine($"Array 'oddNumber' has more uppercase letters ({UpperCharCount(oddNumber)}) then array 'evenNumber' ({UpperCharCount(evenNumber)})");
+                }
+                else
+                {
+                    Console.WriteLine("Array 'oddNumber' and array 'evenNumber'  have an equal number uppercase letters");
+                }
+
+                Console.WriteLine($"{Environment.NewLine}Array 'evenNumber':");
+                PrintArray(evenNumber);
+
+                Console.WriteLine($"{Environment.NewLine}Array 'oddNumber':");
+                PrintArray(oddNumber);
+             }
+            else
+            {
+                Console.WriteLine("You input not integer number or number less than 1");
+            }
+        }
+
+        public static void InputRandomElem(int[] massive)
+        {
+            for (int i = 0; i < massive.Length; i++)
+            {
+                massive[i] = new Random().Next(1, 26);
+            }
+        }
+
+        public static int CountEven(int[] massive)
+        {
+            int count = 0;
+            foreach (var i in massive)
+            {
+                if (i % 2 == 0)
+                {
+                    count++;
+                }
+            }
+
+            return count;
+        }
+
+        public static string IntToEngAlphabet(int numb)
+        {
+            string upperList = "aeidhj";
+            int asciiCode = 'a' + numb - 1;
+            char myChar = (char)asciiCode;
+            string myString = upperList.Contains(myChar.ToString()) ? myChar.ToString().ToUpper() : myChar.ToString().ToLower();
```

Comment:
I think we can return **char** from this method

Reply:
Where ?
string myString = myChar.ToString()

Reply:
char upperChar = upperList.Contains(myChar.ToString()) 
                       ? myChar.ToUpper() 
                       :myChar;

and then return cha instead of string